markdown/3.0/en/ff-asn.md

```diff
@@ -1,224 +1,296 @@
 ## Using the After Search Navigation
-You can use the ASN by adding just one line to your HTML code.
+You can use the ASN by adding one line to your HTML code.
 ```html
 <ff-asn></ff-asn>
 ```
-If you do so, the ASN will look pretty much like nothing. So let's style it.
+Doing so will cause the ASN to use default HTML templates for all filters. You can customize the ASN's appearance by providing your own templates for its elements.
+
 
 ## Adding a filter group
 ```html
 <ff-asn>
     <ff-asn-group></ff-asn-group>
 </ff-asn>
 ```
-Adding a bare `ff-asn-group` does not change the styles but it lets us
-control how different kinds of `ff-asn-groups` should look.
+To customize filter groups you add `ff-asn-group` elements. However, adding an empty `ff-asn-group` as in the example above has no effect. You will likely want to specify your own HTML template.
```

Comment:
see comment above

Reply:
There are several examples involving `ff-asn-group`. Even though this very example has no effect, this step is required by all following examples. It is the entry point if you will. Kind of to convey the basic mechanics of adding a group.
I guess the phrase "has no effect" will make the reader naturally keep on reading.
But maybe I found a suitable wording.

Reply:
```suggestion
To customize filter groups you add `ff-asn-group` elements. However, adding an empty `ff-asn-group` as in the example above has no effect. You will likely want to specify your own HTML template as shown in the following sections.
```

Reply:
mh, I would be happier with removing this example _without effect_ and just state that you wrap your custom templates within `ff-asn-group` as suggested above. But I think your solution is suitable. So feel free to decide